OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -53,7 +53,8 @@ internal static void ProcessOrder(OrderManager orderManager, World world, int cl
 				case "DisableChatEntry":
 					{
 						// Order must originate from the server
-						if (clientId != 0)
+						// Don't disable chat in replays
+						if (clientId != 0 || world.IsReplay)
```

Comment:
This immediately crashes for me with an NRE because world is null. (Use `orderManager.World`?)

Reply:
Apparently world is null when in lobby. See line 95